node/core/bitfield-signing/src/lib.rs

```diff
@@ -218,14 +218,15 @@ impl JobTrait for BitfieldSigningJob {
 	#[tracing::instrument(skip(keystore, metrics, _receiver, sender), fields(subsystem = LOG_TARGET))]
 	fn run(
 		relay_parent: Hash,
+		span: Arc<JaegerSpan>,
```

Comment:
Likewise, I think it likely that the span can be skipped from the tracing here without sacrificing useful debug information.